pkg/api/telemetry.go

```diff
@@ -143,10 +143,17 @@ func NewTelemetryEventFromKubernetesJobEvent(event *pkg.JobEvent, logs string, p
 		Timestamp: strconv.FormatInt(eventTime.Unix(), 10),
 	}
 
-	if env := pkg.NewCronitorConfigParser(cronjob).GetEnvironment(); env != "" {
+	cronitorConfigParser := pkg.NewCronitorConfigParser(cronjob)
+
+	if env := cronitorConfigParser.GetEnvironment(); env != "" {
 		telemetryEvent.Env = env
 	}
 
+	if ok, _ := cronitorConfigParser.EnabledAutoComplete(); !ok && telemetryEvent.Event == Complete {
```

Comment:
please add a code comment here referencing the original issue #47 and explaining why it's writing the event as type `Log`.